modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -9,6 +9,10 @@ using gapi_GNetPackage = cv::gapi::GNetPackage;
 using gapi_ie_PyParams = cv::gapi::ie::PyParams;
 using gapi_wip_IStreamSource_Ptr = cv::Ptr<cv::gapi::wip::IStreamSource>;
 
+// NB: Python wrapper generate T_U for T<U>
+// This behavior is only observed for inputs
+using GOpaque_Size = cv::GOpaque<cv::Size>;
```

Comment:
Should it be fixed on python code generator side ?

Reply:
Oh I see what do you mean by using `using`. Does this behavior blocks us?

Reply:
No, it can be fixed like this